src/sidebar/util/test/watch-test.js

```diff
@@ -0,0 +1,82 @@
+import { createStore } from 'redux';
+
+import { watch } from '../watch';
+
+function counterReducer(state = { a: 0, b: 0 }, action) {
+  switch (action.type) {
+    case 'increment-a':
+      return { ...state, a: state.a + 1 };
+    case 'increment-b':
+      return { ...state, b: state.b + 1 };
+    default:
+      return state;
+  }
+}
+
+describe('sidebar/util/watch', () => {
+  /**
+   * Create a Redux store as a data source for testing the `watch` function.
+   */
+  function counterStore() {
+    return createStore(counterReducer);
+  }
+
+  describe('watch', () => {
```

Comment:
I think it would be informative to demonstrate a test with an object with multiple keys rather than just `a` or `b` 
e.g.

```
watch(store.subscribe, () => store.getState(), callback);
store.dispatch({ type: 'increment-a' });
assert.calledWith(callback, {a: 0, b: 0}, {a: 1, b: 0});

```